packages/dd-trace/test/appsec/rasp/command_injection.express.plugin.spec.js

```diff
@@ -0,0 +1,432 @@
+'use strict'
+
+const agent = require('../../plugins/agent')
+const appsec = require('../../../src/appsec')
+const Config = require('../../../src/config')
+const path = require('path')
+const Axios = require('axios')
+const { getWebSpan, checkRaspExecutedAndHasThreat, checkRaspExecutedAndNotThreat } = require('./utils')
+const { assert } = require('chai')
+
+describe('RASP - command_injection', () => {
+  withVersions('express', 'express', expressVersion => {
+    let app, server, axios
+
+    async function testBlockingRequest () {
+      try {
+        await axios.get('/?dir=$(cat /etc/passwd 1>%262 ; echo .)')
```

Comment:
`%26` is `&`, without encoding it axios was understanding that we had 2 params: `dir=$(cat /etc/passwd 1>` and `2 ; echo .)` (without value)